# 机器人间交互数据

Header header

# 内容ID
int32 cmdID
# 发送者/接受者ID
int32 id
# 数据
uint8[] data